cmd/vm_firewall.go

```diff
@@ -35,6 +35,7 @@ var vmFirewallSetCmd = &cobra.Command{
 			return err
 		}
 
+		defer printVirtualMachineSecurityGroups(vm)
 		return setVirtualMachineSecurityGroups(vm, sgs)
```

Comment:
it'd be more idiomatic to avoid `defer` in this use case.

```go
if err != setVirtualMachineSecurityGroups(vm, sgs); err != nil {
  return err
}

printVirtualMachineSecurityGroups(vm)
```

Reply:
Done.